npm/iptm/iptm_test.go

```diff
@@ -146,14 +117,90 @@ func TestInitNpmChains(t *testing.T) {
 }
 
 func TestUninitNpmChains(t *testing.T) {
-	calls := unInitCalls
-
-	fexec := testutils.GetFakeExecWithScripts(calls)
-	defer testutils.VerifyCalls(t, fexec, calls)
-	iptMgr := NewIptablesManager(fexec, NewFakeIptOperationShim(), util.PlaceAzureChainAfterKubeServices)
-
-	if err := iptMgr.UninitNpmChains(); err != nil {
-		t.Errorf("TestUninitNpmChains @ iptMgr.UninitNpmChains")
+	tests := []struct {
+		name  string
+		calls []testutils.TestCmd
+	}{
+		{
+			name: "no v2 npm chains exist",
+			calls: []testutils.TestCmd{
```

Comment:
Can we convert this to a helper function somewhere. It would be useful in other tests as well IMO.

Reply:
perhaps in another PR to clean up UTs? There are a couple areas of improvement